data/mapcss-mapping.csv

```diff
@@ -213,8 +213,8 @@ railway|rail|tunnel;[railway=rail][tunnel?];;name;int_name;212;
 man_made|wastewater_plant;213;
 man_made|flagpole;214;
 place|county;215;
-deprecated|deprecated;216;x
-deprecated|deprecated;217;x
+amenity|vending_machine|coffee;[amenity=vending_machine][vending=coffee];;name;int_name;216;
+amenity|vending_machine|sweets;[amenity=vending_machine][vending=sweets];;name;int_name;217;
```

Comment:
Less value (216, 217) means more frequent tag in OSM. Please, put this types somewhere around existing amenity=vending_machine.

Reply:
Here will be ok:
deprecated|deprecated;1061;x
deprecated|deprecated;1062;x

Reply:
> Less value (216, 217) means more frequent tag in OSM. Please, put this types somewhere around existing amenity=vending_machine.

Wow, I didn't know that either, thanks!
Does it have any consequences e.g. performance- or space-wise?

Reply:
Well, yes in general, but not always.